### PR TITLE
[CI] Extract a set of commands that are needed to fill dependency.

### PR DESCRIPTION
**Pull request by 130s**
The portion of codes can be reusable, so worth making them self-contained.

### CODE REVIEW

**Comment by 130s** [comment]
Looks like I based off from the origin's head. I'll re-base off from POR's fork.

**Comment by 130s** [comment]
@geoffreychiou Please review.

BTW @shaun-edwards 1) Please create POR specific branch and set as a default. I believe our convention has been something like `por_%ORIGIN_DEFAULTBRANCH%` (so could be `por_development` for this repo). Having our default branch with the same name with the one at the origin, it's very cumbersome to create a PR that goes to our fork. 2) Can I get a maintainer access on this repo?

**Comment by 130s** [comment]
Btw this should be suggested to the upstream once it's proven to work.

**Comment by geoffreychiou** [comment]
+1.

File: .travis.yml [hunk 1]
```diff
@@ -3,12 +3,7 @@ dist: xenial
 
   # - git clone -v --progress https://github.com/doronhi/realsense.git  # This is Done automatically by TravisCI
 before_install:
-  - echo 'deb http://realsense-hw-public.s3.amazonaws.com/Debian/apt-repo xenial main' || sudo tee /etc/apt/sources.list.d/realsense-public.list
-  # - sudo apt-key adv --keyserver keys.gnupg.net --recv-key C8B3A55A6F3EFCDE || sudo apt-key adv --keyserver hkp://keys.gnupg.net:80 --recv-key C8B3A55A6F3EFCDE
-  - sudo add-apt-repository "deb http://realsense-hw-public.s3.amazonaws.com/Debian/apt-repo xenial main"
-  - sudo apt-get update -qq
-  - sudo apt-get install librealsense2-dkms --allow-unauthenticated -y 
-  - sudo apt-get install librealsense2-dev --allow-unauthenticated -y
+  - .install_dependency.sh
```
Comment (geoffreychiou) [inline .travis.yml]: Does this need to be `- ./.install_dependency.sh` ?

Comment (130s) [reply .travis.yml]: That sounds right...updated.

#NoCI-NoCompany.

**Review by geoffreychiou — APPROVED** [review]
Looks great. I don't have merge rights here so we might need @shaun-edwards.

**Pull request merged**